tests/integration/mount_path_test.go

```diff
@@ -92,18 +92,19 @@ func (s *MountPathSuite) SetupSuite() {
 	}
 
 	// Create folders over smbclient
-	smbclient := smbclient.MustPodExec(s.tc, testNamespace,
+	smbClient := smbclient.MustPodExec(s.tc, testNamespace,
```

Comment:
s/smbClient/client ?

I think we used _client_ in recent changes which is  also less confusing.

Reply:
Sure.  I often have trouble being consistent with naming. Possibly, because I'm never truly happy by my past choices. ;-)

Reply:
smbclient vs smbClient is like potatoes vs potatoes, isn't it? ;)